tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -78,9 +78,9 @@ $ sudo pip3 install --upgrade https://storage.googleapis.com/tensorflow/mac/tens
 ```
 
 
-You can now [test your installation](#test_install).
+You can now [test your installation](#test-the-tensorflow-installation).
 
-## Virtualenv installation {#virtualenv_install}
```

Comment:
Good to know. If we want to make more of these changes to the rest of the non-auto-generated Markdown, I can modify the script I used in #584 to do so.